deploy/kubernetes/overlays/prow-gke-head/kustomization.yaml

```diff
@@ -1,5 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 bases:
 - ../../base
-patches:
-- controller_images.yaml
-- node_images.yaml
+images:
+- name: REPLACEME/gcp-compute-persistent-disk-csi-driver
+  newName: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver
+  newTag: "latest"
```

Comment:
would it make more sense to rename this overlay `prow-gke-release-staging` or something like that?

Reply:
wait nevermind, thats another overlay. `prow-gke-release-staging-latest`?

Reply:
renamed all prow-gke overlays